sky/packages/sky/lib/src/rendering/binding.dart

```diff
@@ -92,10 +92,12 @@ class _UiEventConverter {
         break;
     }
 
+    int pointer = (state == null) ? event.pointer : state.pointer;
```

Comment:
Aren't you mixing two namespaces here? (The OS pointer IDs and the Flutter pointer IDs?)

I think we shouldn't be even receiving hover events unless we also implement pointeradd/pointerremove.